### PR TITLE
tickets/DM-5005

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/validate/drp/validate.py [hunk 1]
```diff
@@ -37,6 +37,19 @@
 from .util import getCcdKeyName, repoNameToPrefix, calcOrNone
 from .io import saveKpmToJson
 
+from lsst.obs.cfht import MegacamMapper
```
Comment (wmwv) [inline python/lsst/validate/drp/validate.py:40]: Sorry, @r-owen I didn't catch this in my earlier review!  I'm still developing my reviewing skills.

`validate_drp` does not depend on `obs_cfht`.  We can't assume that this mapper is available.

`objIdToCcdId` should be passed the mapper in some way.

[I totally understand how this routine is useful to you.]

[Sorry, I see where it's used.  But it does seem out of scope for DM-5005.]

**Comment by r-owen** [comment]
This was an accident. I apologize. It was only meant to be temporary code for my own debugging purposes. I removed it on master.
